Makefile

```diff
@@ -91,7 +91,10 @@ package_statistics_collector: build_statistics_collector package_dir_structure
 	cd $(CC_BUILD_DIR) && \
 	ln -sf ../lib/python3/codechecker_statistics_collector/cli.py bin/post-process-stats
 
-package: package_dir_structure set_git_commit_template package_plist_to_html package_tu_collector package_report_converter package_report_hash package_merge_clang_extdef_mappings package_statistics_collector
+package_gerrit_skiplist:
+	cp -rp scripts/gerrit_changed_files_to_skipfile.py $(CC_BUILD_DIR)/bin
```

Comment:
If we specify the interpreter in the python file below we can copy it without the .py extension so this command can be called easily.
Also other commands in the bin directory doesn't have extension so it would be more consistent and the user doesn't have to call this script through the python binary.

Reply:
Yes, the same reason I did not this change what I described in my previous answer. I thought about it but the script is python2/python3 compatible now, with that change we limit it to python3 only do we want that?